react-native/react/engine/rpc.js

```diff
@@ -4,10 +4,10 @@ import EngineError from './errors'
 import rpc from 'framed-msgpack-rpc'
 
 const {
-  transport: { Transport: RpcTransport }
+  transport: { RobustTransport }
```

Comment:
Does no other part of the code need to be changed for this change from the key `Transport` -> `RobustTransport`?